int_experian/int_experian_sgjc/cartridge/static/default/js/edqUtils.js

```diff
@@ -484,6 +484,11 @@ function edqSetProWebConfiguration() {
 			elements: ["ZIP Code", "+4 code"],
 			modalFieldSelector: "#interaction-address--original-postal-code",
 		},
+		{
+			field: edqCountryLineSelector,
+			elements: ['Two character ISO country code'],
+			modalFieldSelector: "#interaction-address--original-country-code",
```

Comment:
As far as I know this modalField selector doesn't actually exist - in this case we usually just make it a hash, e.g. `modalFieldSelector: '#'`.

Reply:
I did the change to fit the example